docs/sources/configure-client/trace-span-profiles/java-span-profiles.md

```diff
@@ -2,7 +2,7 @@
 title: Span profiles with Traces to profiles for Java
 menuTitle: Span profiles with Traces to profiles (Java)
 description: Learn about and configure Span profiles with Traces to profiles in Grafana for the Java language.
-weight: 100
+weight: 101
 ---
 
 # Span profiles with Traces to profiles for Java
```

Comment:
I'd add the same introduction that we have for the .NET page here. I haven't copied it here yet because of the suggestions about the OpenTelemetry section.

Reply:
If you like the suggestions  above, then just copy and paste the same introduction here (making any necessary changes for differences between .NET and Java).